assets/js/components/Layout/Layout.jsx

```diff
@@ -10,10 +10,10 @@ import {
   EOS_STORAGE,
   EOS_LIST,
   EOS_SETTINGS,
+  EOS_KEYBOARD_DOUBLE_ARROW_LEFT,
```

Comment:
Dude, good catcth!

Reply:
Yeah he's been waiting for days for this :smile:

Reply:
Both EOS_KEYBOARD_DOUBLE_ARROW_RIGHT and EOS_KEYBOARD_DOUBLE_ARROW_LEFT are now being imported.